sections/05_treat.md

```diff
@@ -208,54 +208,52 @@ which small molecules (also referred to as chemical compounds or ligands)
-relationships in the molecular graph, and more sophisticated derived properties
-[@doi:10.1002/9783527628766].   Alternatively, chemicals can be characterized
-with the fingerprint bit vectors, textual strings, or novel learned
-representations described below. Neural networks have a long history in this
-domain [@tag:Baskin2015_drug_disc] `TODO: can add additional references besides
-this review`, and the 2012 Merck Molecular Activity Challenge on Kaggle
-generated substantial excitement about the potential for high-parameter deep
-learning approaches.  The winning submission was an ensemble that included a
-multitask multilayer perceptron network [@tag:Dahl2014_multi_qsar], and the
-Merck sponsors noted drastic improvements over a random forest (RF) baseline,
-remarking "we have seldom seen any method in the past 10 years that could
-consistently outperform RF by such a margin" [@tag:Ma2015_qsar_merck].
-Subsequent work explored the effects of jointly modeling far more targets than
-the Merck challenge [@tag:Unterthiner2014_screening
-@tag:Ramsundar2015_multitask_drug], with [@tag:Ramsundar2015_multitask_drug]
-showing that the benefits of multi-task networks had not yet saturated even with
-259 targets.  Although a deep learning approach, DeepTox
-[@tag:Mayr2016_deep_tox], was also the overall winner of another competition,
-the Toxicology in the 21st Century (Tox21) Data Challenge, it did not dominate
-alternative methods as thoroughly as in other domains.  DeepTox was the top
-performer on 9 of 15 targets and highly competitive with the top performer on
-the others.  However, for many targets there was little separation between the
-top two or three methods.  A reliance on AUC ROC `TODO: define here?` for the
-evaluation (see Discussion) further hinders the ability to declare Tox21 as an
-outright success for deep learning.
+features without requiring prior knowledge of the target, as opposed to
+strategies specifically modeling features such as protein structure `TODO: add
+examples`. Chemical features may be represented as a list of molecular
+descriptors such as molecular weight, atom counts, charge representations,
+summaries of atom-atom relationships in the molecular graph, and more
+sophisticated derived properties [@doi:10.1002/9783527628766].   Alternatively,
+chemicals can be characterized with the fingerprint bit vectors, textual
+strings, or novel learned representations described below. Neural networks have
+a long history in this domain [@tag:Baskin2015_drug_disc] `TODO: can add
+additional references besides this review`, and the 2012 Merck Molecular
+Activity Challenge on Kaggle generated substantial excitement about the
+potential for high-parameter deep learning approaches.  The winning submission
+was an ensemble that included a multitask multilayer perceptron network
+[@tag:Dahl2014_multi_qsar], and the Merck sponsors noted drastic improvements
```

Comment:
I prefer to keep it following literature in this domain.  Subsequent papers (e.g. #251) refer to it using the Merck name:

> The first foray of deep learning into QSAR was the Merck challenge in 2012